js/src/logger.ts

```diff
@@ -899,6 +1102,13 @@ export function currentExperiment(): Experiment | undefined {
   return _state.currentExperiment.getStore();
 }
 
+/**
+ * Returns the currently-active logger (set by `braintrust.withLogger` or `braintrust.withCurrent`). Returns undefined if no current logger has been set.
```

Comment:
There is no `withLogger` function. Do we want one?

Reply:
added